index.html

```diff
@@ -116,22 +135,24 @@ <h3 class="skill-title">Preprocessors</h3>
         </a>
       </section>
       <!-- Internal link to the projects -->
-      <a href="index.html#projects" id="to-projects">
+      <a href="index.html#projects" class="to-projects">
         <svg width="20" height="24" viewBox="0 0 19 23" fill="none" xmlns="http://www.w3.org/2000/svg">
           <path d="M0 12.8005L9.15 21.6505L18 12.8005" stroke="#C4C4C4"/>
           <path d="M9.1499 0.500488V21.8005" stroke="#C4C4C4"/>
         </svg>
         <span>Work</span>
       </a>
+      </section>
       <!-- My projects section -->
       <section class="projects-section">
         <h2 class="projects-title">My recent work</h2>
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using the info from the Figma templates. The desktop version has the 'My recent work' capitalized only the first letter, and not all letters like the mobile version. Please fix that😊

Reply:
Sure, let me fix this one 😄